Core/StringHelper.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Core
+{
+    public static class StringHelper
+    {
+        public static int GetStableHashCode(this string str)
+        {
+            unchecked
+            {
+                var hash1 = 5381;
+                var hash2 = hash1;
+
+                for(var i = 0; i < str.Length && str[i] != '\0'; i += 2)
+                {
+                    hash1 = ((hash1 << 5) + hash1) ^ str[i];
+                    if (i == str.Length - 1 || str[i+1] == '\0')
+                        break;
```

Comment:
Codacy found an issue: [Refactor the code in order to remove this break statement.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=6077033)